composer.json

```
@@ -106,7 +106,7 @@
       "rm -rf cx-api-spec"
     ],
     "box-install": [
-      "curl -L https://github.com/humbug/box/releases/latest/download/box.phar -o build/box.phar"
+      "curl -f -L https://github.com/box-project/box/releases/download/3.11.1/box.phar -o build/box.phar"
```

```suggestion
      "wget -O build/box.phar https://github.com/box-project/box/releases/download/3.11.1/box.phar"
```

Wget will set exist status to 8 when it is not an HTTP/404.

---

Thanks! But I think we'll stick with curl since `-f` should have the same effect, and we know it's available everywhere in our infrastructure.